test/test-raja-perf-suite.cpp

```diff
@@ -0,0 +1,129 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-22, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "gtest/gtest.h"
+
+#include "common/Executor.hpp"
+#include "common/KernelBase.hpp"
+
+#include <vector>
+#include <string>
+#include <iostream>
+#include <cmath>
+
+TEST(ShortSuiteTest, Basic)
+{
+
+// Assemble command line args for basic test
+  int argc = 4;
+#if ( (defined(RAJA_ENABLE_HIP) && \
+       (HIP_VERSION_MAJOR < 5 || \
+       (HIP_VERSION_MAJOR == 5 && HIP_VERSION_MINOR < 1)) ) \
+      || (defined(RAJA_COMPILER_CLANG) && __clang_major__ == 11) )
+  argc = 6;
+#endif
```

Comment:
I suppose these wouldn't overlap but should they still be separated to better match up with what is below?

Reply:
Change pushed.